jmx-scraper/src/main/resources/hadoop.yaml

```diff
@@ -0,0 +1,61 @@
+---
+rules:
+  - bean: Hadoop:service=NameNode,name=FSNamesystem
+    prefix: hadoop.name_node.
+    metricAttribute:
+      node_name: beanattr(tag\.Hostname)
+    mapping:
+      CapacityUsed:
+        metric: capacity.usage
+        type: updowncounter
+        unit: by
```

Comment:
There were some recent discussions about what units we need align with semconv. The agreement is that we should fix all units that do not align with [semconv](https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units)
Because of this please update `by` to `By`